composer.json

```diff
@@ -133,6 +133,7 @@
         "npm-asset/jquery.caret": "^0.3.1",
         "npm-asset/diff": "^3.5",
         "npm-asset/photoswipe": "^4.1.2",
-        "npm-asset/shariff": "^3.0.1"
+        "npm-asset/shariff": "^3.0.1",
+        "drupal/geocoder": "^2.0@beta"
```

Comment:
Only there for testing purposes, part of the composer.json and dependencies of the module :) but easier install for us :D